dotnet build keeps prompting NU1006 warning

I'm running on dotnet `1.0.1.001032`.

Even after `dotnet restore`, I keep getting following warnings. They don't fail the build.
```
Project OmniSharp.Abstractions will be compiled because some of its inputs were newer than its oldest output.
Compiling OmniSharp.Abstractions for DNX,Version=v4.5.1
C:\code\omnisharp-roslyn\src\OmniSharp.Abstractions\project.lock.json(1,0): warning NU1006: Dependencies in project.json were modified. Please run "dotnet restore" to generate a new lock file.
```

Here's the project.json file: (the Lock file is too big to be copied here. I can provide it offline):
```
{
  "version": "1.0.0-*",
  "compilationOptions": {
    "warningsAsErrors": true
  },
  "dependencies": {
    "Newtonsoft.Json": "7.0.1",
    "Microsoft.CodeAnalysis": "1.1.1",
    "Microsoft.Composition": "1.0.30",
    "Microsoft.Extensions.Caching.Memory": "1.0.0-rc2-16050",
    "Microsoft.Extensions.Configuration": "1.0.0-rc2-16014",
    "Microsoft.Extensions.Configuration.Binder": "1.0.0-rc2-16014",
    "Microsoft.Extensions.Logging.Console": "1.0.0-rc2-16023",
    "Microsoft.Extensions.FileSystemGlobbing": "1.0.0-rc2-15983",
    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-rc2-16532"
  },
  "frameworks": {
    "dnx451": {
      "frameworkAssemblies": {
        "System.Runtime": "",
        "System.Threading": "",
        "System.Threading.Tasks": "",
        "System.Collections": "",
        "System.Linq": "",
        "System.Text.Encoding": "",
        "System.IO": "",
        "System.Reflection": "",
        "System.Security": "",
        "System.Globalization": "",
        "System.Xml": "",
        "System.Xml.Linq": ""
      }
    },
    "dnxcore50": {
      "imports": "portable-net45+win8",
      "dependencies": {
        "NETStandard.Library": "1.0.0-rc2-23720",
        "System.IO.FileSystem.Watcher": "4.0.0-rc2-23720",
        "System.Threading.Thread": "4.0.0-rc2-23720"
      }
    }
  }
}
```

--- a/src/Microsoft.DotNet.ProjectModel/Graph/LockFile.cs
+++ b/src/Microsoft.DotNet.ProjectModel/Graph/LockFile.cs
@@ -60,7 +60,9 @@ namespace Microsoft.DotNet.ProjectModel.Graph
                 // If the framework name is empty, the associated dependencies are shared by all frameworks
                 if (group.FrameworkName == null)
                 {
-                    actualDependencies = project.Dependencies.Select(RenderDependency).OrderBy(x => x);
+                    actualDependencies = project.Dependencies
+                        .Select(RenderDependency)
+                        .OrderBy(x => x, StringComparer.OrdinalIgnoreCase);
                 }
                 else
                 {
@@ -71,7 +73,9 @@ namespace Microsoft.DotNet.ProjectModel.Graph
                         return false;
                     }
 
-                    actualDependencies = framework.Dependencies.Select(RenderDependency).OrderBy(x => x);
+                    actualDependencies = framework.Dependencies
+                        .Select(RenderDependency)
+                        .OrderBy(x => x, StringComparer.OrdinalIgnoreCase);
                 }
 
                 if (!actualDependencies.SequenceEqual(expectedDependencies))
@@ -84,16 +88,6 @@ namespace Microsoft.DotNet.ProjectModel.Graph
             return true;
         }
 
-        private string RenderDependency(LibraryRange arg)
-        {
-            var name = arg.Name;
-
-            if (arg.Target == LibraryType.ReferenceAssembly)
-            {
-                name = $"fx/{name}";
-            }
-
-            return $"{name} {VersionUtility.RenderVersion(arg.VersionRange)}";
-        }
+        private string RenderDependency(LibraryRange arg) => $"{arg.Name} {VersionUtility.RenderVersion(arg.VersionRange)}";
     }
 }

